main.go

```diff
@@ -49,13 +49,20 @@ func main() {
 		level.Error(logger).Log("err", "-token is required")
 		os.Exit(1)
 	}
-	if len(serviceIDs) <= 0 {
-		level.Error(logger).Log("err", "at least one -service ID is required")
-		os.Exit(1)
-	}
 
 	level.Debug(logger).Log("msg", "looking up service names")
 	serviceNames := getServiceNames(*token, serviceIDs, log.With(logger, "query", "api.fastly.com"))
+	if len(serviceNames) == 0 {
+		level.Error(logger).Log("err", "service name lookup error, retrying")
+		for i := 1; len(serviceNames) == 0; i++ {
```

Comment:
Nah, just exit right away. Anyone who cares will be running this thing in a supervisor.